test/test_load_nifti.py

```diff
@@ -0,0 +1,214 @@
+#!usr/bin/env python3
```

Comment:
i would suggest to wait for the dev core team to agree on the proposed developments before implementing the unit tests (otherwise, the unit tests might become stale/irrelevant)

Reply:
Sure, I implemented them for now so I can validate that the core functionality of the function works correctly, but I will wait for further information.